### PR TITLE
OSSMDOC-454: Release Notes for distributed tracing 2.1.0

### PR DESCRIPTION
**Pull request by JStickler**
Preview is here -> https://deploy-preview-40987--osdocs.netlify.app/openshift-enterprise/latest/distr_tracing/distributed-tracing-release-notes.html

Eng review - rkukura
QE review - iblancasa
Peer review - lpettyjo

### CODE REVIEW

**Review by rkukura — APPROVED** [review]

File: modules/distr-tracing-rn-fixed-issues.adoc [hunk 1]
```diff
@@ -14,6 +14,8 @@ Fix - What did we change to fix the problem?
 Result - How has the behavior changed as a result? Try to avoid “It is fixed” or “The issue is resolved” or “The error no longer presents”.
 ////
 
+* link:https://issues.redhat.com/browse/TRACING-2340[TRACING-2340] Fixed an issue where the jaeger-query UI login failed with the following error message: `oauthproxy.go:445: ErrorPage 500 Internal Error configmap "oauth-serving-cert" not found`. This prevented the user from accessing the Jaeger console. On startup the {JaegerName} Operator looks for the oauth-proxy image from the OpenShift cluster, and if not found it uses the image supplied in the arguments. The Operator has been updated to provide the missing arguments.
```
Comment (iblancasa) [inline modules/distr-tracing-rn-fixed-issues.adoc:17]: Should we include this issue in the release notes? It was introduced and solved in this version. So, AFAIK, there are not published releases with that issue, right?

**Review by iblancasa — APPROVED** [review]

File: modules/distr-tracing-rn-fixed-issues.adoc (hunk 1 above)
Comment (JStickler) [reply modules/distr-tracing-rn-fixed-issues.adoc]: Good point.  I'll remove it.

File: modules/distr-tracing-rn-technology-preview.adoc [hunk 2]
```diff
@@ -17,6 +17,35 @@ Technology Preview features are not supported with Red Hat production service le
 These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process. For more information about the support scope of Red Hat Technology Preview features, see https://access.redhat.com/support/offerings/techpreview/.
 ====
 
+== {DTProductName} 2.1.0 Technology Preview
+
+This release introduces a breaking change to how to configure certificates in the OpenTelemetry custom resource file.  In the new version, the `ca_file` moves under `tls` in the custom resource, as shown in the two examples below.
```
Comment (lpettyjo) [inline modules/distr-tracing-rn-technology-preview.adoc:22]: Looks like there may be a double space between the first and second sentence. 

IBM SG: Don't use "below" to indicate a relative location in a document, as in "the information below". Use "the information later in this section". Alternatively, you can use "following" to refer to a table, to art, or to a list. Example: "The following table shows ..."

**Review by lpettyjo — COMMENTED** [review]
Otherwise, LGTM!

File: modules/distr-tracing-rn-technology-preview.adoc (hunk 2 above)
Comment (JStickler) [reply modules/distr-tracing-rn-technology-preview.adoc]: Fixed.

**Pull request merged**